src/test/java/uk/gov/ons/ctp/response/sample/service/impl/SampleServiceImplTest.java

```diff
@@ -292,30 +297,25 @@ public void testUploadInvalidTypeSample() throws Exception {
     when(this.sampleSvcStateTransitionManager.transition(SampleState.INIT, SampleEvent.FAIL_VALIDATION))
             .thenReturn(SampleState.FAILED);
     // Given
-    MockMultipartFile file = new MockMultipartFile("file", "data".getBytes());
+    MockMultipartFile file = createMockMultipartFile();
 
     final String invalidType = "invalid-type";
 
     SampleSummary summary = createSampleSummary(5, 10);
     when(csvIngesterBusiness.ingest(any(SampleSummary.class), any(MultipartFile.class))).thenReturn(summary);
 
     // When
-    SampleSummary finalSummary = sampleServiceImpl.ingest(summary, file, invalidType);
+    sampleServiceImpl.ingest(summary, file, invalidType);
   }
 
   @Test
-  public void testIngestMessagesSent() throws Exception {
```

Comment:
Is this a valid test that is being deleted?

Reply:
The messages are no longer sent from the SampleServiceImpl